main.go

```diff
@@ -694,6 +750,7 @@ func main() {
 	router.GET("/managementPlatform", a.ManagementPlatform)
 	router.GET("/clusterInfo", a.ClusterInfo)
 	router.GET("/containerUptimes", a.ContainerUptimes)
+	router.GET("/aggregatedPrices", a.AggregateCostModel)
```

Comment:
proposal: can we make this `/aggregatedCostModel` for a little more consistency?